src/utils/getViews.js

```diff
@@ -0,0 +1,6 @@
+export const getViews = (views) => {
```

Comment:
I liked how you used to convert number views into k and m format

Reply:
> well implemented redux, and also you created a separate file for service calls . A little bit of UI is not fixed as Roshan suggested. Rest is good

Surprisingly, I don't see any UI issue on my end.

Thanks for the review @sushil-mod